public/building.html

```diff
@@ -0,0 +1,124 @@
+  </head>
+
+  <body>
+
+    <nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container-fluid">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="index.html">Powell County Schools</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav">
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Facilities <span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li class="active"><a href="building.html">Building</a></li>
+                <li><a href="room.html">Room</a></li>
+              </ul>
+            </li>
+            <li><a href="#">Inventory</a></li>
+            <li><a href="#">Check-In</a></li>
+            <li><a href="#">Reports</a></li>
+          </ul>
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="login.html">Log out</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-3 col-md-2 sidebar">
+          <ul class="nav nav-sidebar">
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Facilities <span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li class="active"><a href="building.html">Building</a></li>
+                <li><a href="room.html">Room</a></li>
+              </ul>
+            </li>
+            <li><a href="#">Inventory</a></li>
+            <li><a href="#">Check-In</a></li>
+            <li><a href="#">Reports</a></li>
+          </ul>
+          <ul class="nav nav-sidebar">
+            <li><a href="login.html">Log out</a></li>
+          </ul>
+        </div>
+        <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+          <h1 class="page-header">Manage Building</h1>
+          <h4>Building</h4>
+          <div class="bs-docs-example">
+            <input type="text" class="span3" style="margin: 0 auto;" data-provide="typeahead" data-items="4" data-source='["Highland Elementary","Powell County High School"]'>
+          </div>
+          <br class="divider">
+          <p>
+            <a class="btn btn-primary active" href="#" role="button">Create</a>
+            <a class="btn btn-primary" href="#" role="button">Edit</a>
+            <a class="btn btn-primary" href="#" role="button">Delete</a>
```

Comment:
Having the "Delete" button here is a bit confusing. The way we see this page working, you search for a building and once you select one, the lower part of the page gets populated with that building's  details. The lower part of the page becomes the workspace for dealing with an individual building. It would be more appropriate for the delete button to be there.